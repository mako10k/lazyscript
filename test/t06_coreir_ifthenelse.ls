(~ifthenelse (~true) (1) (2));
