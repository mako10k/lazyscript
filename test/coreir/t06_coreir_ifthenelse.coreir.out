; CoreIR dump (temporary)
(if (var true) (int 1) (int 2))
